### ISSUE TITLE
added captcha handle with unicaps

### ISSUE THREAD

**Issue by tjengbudi** [opsdisk/yagooglesearch#17]
hello is there any chance to added captcha handle with unicaps library?
it might nice to have alternative to escaped blocked  by google

what you think about this?

**Comment by opsdisk** [comment]
Hi @tjengbudi - Although using something like https://github.com/sergey-scat/unicaps to solve captchas is possible, it's not a capability I want to support at this time.  I'd recommend:

* adding logic in your script to handle HTTP 429s (see https://github.com/opsdisk/yagooglesearch#http-429-detection-and-recovery-optional)
* utilizing proxies to spread the search load, see https://github.com/opsdisk/yagooglesearch#http-and-socks5-proxy-support

**Comment by tjengbudi** [comment]
I tried using proxy. However I fetch from free proxy and never get success
to connect.

I already see the 429 error and try with something to change in the
get_page method. But I am not success with that.

Unicaps or something like this is optional. Something really free of use
will be nice. But I also need something really fast to implement to cover
before trying more with proxy and/or others

**Comment by opsdisk** [comment]
Free proxies are abused and probably already blocked by Google, so I'd stay away from them.  Sounds like you have some options to explore/investigate.  I'll keep this issue open for a while if there are any questions around yagooglesearch.

**Comment by tjengbudi** [comment]
Could you recommend proxy service that can used with this?

**Comment by opsdisk** [comment]
In general, you'll have to do it yourself.  You can deploy your own Virtual Private Servers and use ssh -D for SOCKS proxies or leverage something like this https://github.com/Ge0rg3/requests-ip-rotator

**Comment by tjengbudi** [comment]
I try several way.
1: using tor as proxy. However Google always found it and block
2: using Unicaps. First time now working it cause the parameter. I found that your params always used num= it will always got blocked by google
After remove num parameter, captcha can resolved. But when create new instance it will be blocked again. So I use first create instance, and force change the query and update the url. 2query fetch after resolved captcha. Next trial might be saved the cookies

**Comment by opsdisk** [comment]
Closing this one out.
